frontend/asset_mapper.rst

```diff
@@ -413,10 +413,7 @@ the page as ``link`` tags in the order they were imported.
 .. note::
 
     Importing a CSS file is *not* something that is natively supported by
-    JavaScript modules and normally causes an error. AssetMapper makes this
-    work by adding an importmap entry for each CSS file that is valid, but
-    does nothing. AssetMapper adds a ``link`` tag for each CSS file, but when
-    the JavaScript executes the ``import`` statement, nothing additional happens.
+    JavaScript modules. AssetMapper makes this work by adding an importmap entry for each CSS file.
```

Comment:
The original sentence can be improved, but this one feels the CSS entries are "normal" import map ones, when they are not.

Maybe add this precision in your rewording ?